### ISSUE TITLE
[feature request] Multiple path per handler

### ISSUE THREAD

**Issue by vincentsarago** [starlite-api/starlite#44]
I sometimes want to use optional `path` parameter. For this I need to be able to set more than one path per route: 

```python
@get(path="/image")
@get(path="/image.{format:str}")
def get_image(format: str = "jpeg"):
    ...
```
see: https://github.com/developmentseed/titiler/blob/master/src/titiler/core/titiler/core/factory.py#L436-L450

**Comment by Goldziher** [comment]
You can do this alread in the following manner:

```python
def get_image_handler(format: str = "jpeg"):
    ...

get_image = get(path="/image")(get_image_handler)
get_image_by_format = @get(path="/image.{format:str}")(get_image_handler)
```

Its of course not very pretty.

The question is this - should we support multiple paths by doing this:

```python
@get(["path/subpath", "path/anothersubpath"])
```

Or allow multiple decorators. Obviously multiple decorators allow for a variety of other parameters etc. for example - different dependencies and so forth. Yet this is going to be trickier to implement.

Thoughts?

**Comment by Goldziher** [comment]
This one is implemented in #49
